api/app.js

```diff
@@ -84,7 +84,7 @@ app.use('/', indexRouter);
 app.use(['/parent', '/parents'], parentRouter);
 app.use(['/profile', '/profiles'], profileRouter);
 app.use(['/child', '/children'], childRouter);
-app.use(['/storyNew', '/storiesNew'], storyNewRouter);
```

Comment:
This was a safe move. I think it might be worth it to leave a comment in the code as well, so in the future, they'll know to remove '/storyNew' and '/storiesNew' when it's time.